app/templates/components/public/call-for-speakers.hbs

```diff
@@ -20,9 +20,16 @@
 {{sanitize data.speakersCall.announcement}}
 <div class="ui hidden divider"></div>
 {{#if data.speakersCall.isOpen}}
-  <button type="submit" class="ui teal submit button" {{action 'openProposalModal'}}>
-    {{t 'Submit Proposal'}}
-  </button>
+  {{#if session.isAuthenticated}}
```

Comment:
Can we just have a single action?

Reply:
@niranjan94 Combined them as single action. Please review.